rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import typescript from 'rollup-plugin-typescript2';
+import dts from 'rollup-plugin-dts';
+import replace from '@rollup/plugin-replace';
+import pkg from './package.json';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export default [
+    {
+        input: 'src/npm/index.ts',
+        output: [
+            {
+                file: pkg.main,
+            },
+        ],
+        plugins: [
+            typescript({ outfile: 'index.js' }),
+            replace({
+                preventAssignment: true,
+                'process.env.AZURE_GRAPHQL_URI': JSON.stringify(process.env.AZURE_GRAPHQL_URI),
```

Comment:
```suggestion
                'process.env.GRAPHQL_URI': JSON.stringify(process.env.GRAPHQL_URI),
```

Not necessarily hosted on azure.